c10/core/impl/LocalTensorTypeSet.cpp

```diff
@@ -14,29 +14,55 @@ namespace {
 
 // NB: Zero initialized!
 thread_local uint64_t raw_excluded;
+thread_local uint64_t raw_included;
```

Comment:
These are used adjacent to each other in dispatch, so it would be more efficient if they were in a single thread_local struct. Otherwise there is a sequence of like 4 loads to get to each one. I would imagine this struct would grow into the generic thread-local dispatch state that the chain of responsibility would need to call `super` as well.